src/pages/api/export/[tableName].ts

```diff
@@ -24,8 +27,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
   try {
     const { tableName, minlon, minlat, maxlon, maxlat } = params
-    const functionName = 'atlas_export_geojson_' + tableName.toLowerCase()
-    await prismaClientForRawQueries.$queryRawUnsafe('SET search_path TO public')
+    const functionName = exportFunctionIdentifier(tableName.toLowerCase())
```

Comment:
```suggestion
    const functionName = exportFunctionIdentifier(tableName)
```

Das `toLowerCase` passiert im Helper, richtig?